docs/learn-kusama-vs-polkadot.md

```diff
@@ -0,0 +1,57 @@
+---
+id: learn-kusama-vs-polkadot
+title: Comparing Polkadot and Kusama
+sidebar_label: Comparing Polkadot and Kusama
+---
+
+# What is the difference between Polkadot and Kusama?
+
+Although they are built on nearly the same codebase, Polkadot and Kusama are independent, standalone networks with different priorities.  
+Kusama is wild and fast; great for bold experimentation and early-stage deployment. Polkadot is more conservative, prioritizing stability and dependability.
+
+## What the two networks have in common
+
+Kusama was released as an early version of the same code to be used in Polkadot, which means they share the same underlying architecture: a multichain, heterogeneously-sharded design based on nominated proof of stake (NPoS).
+Both networks also share key innovations like on-chain governance, hot-swappable runtimes for forkless, on-chain upgrades, and cross-chain message passing (XCMP) for interoperability.
+Governance on both Polkadot and Kusama is designed to be decentralized and permissionless, giving a say in how the network is run to everyone who owns the native token (DOT for Polkadot, KSM for Kusama). Therefore, **over time the networks will evolve independently, converging or diverging according to the decisions of their respective communities.**
+
+## Key differences
+
+There are a few important distinctions to be made.
+
+![polkadot_vs_kusama](/img/polkadot_vs_kusama.png)
```

Comment:
I would recommend here to place the image inside the `assets` folder. Logan's new img optimizer will be optimizing the images inside that folder.